sdk/nodejs/firestore/v1beta1/index_.ts

```diff
@@ -8,31 +8,31 @@ import * as utilities from "../../utilities";
 /**
  * Creates the specified index. A newly created index's initial state is `CREATING`. On completion of the returned google.longrunning.Operation, the state will be `READY`. If the index already exists, the call will return an `ALREADY_EXISTS` status. During creation, the process could result in an error, in which case the index will move to the `ERROR` state. The process can be recovered by fixing the data that caused the error, removing the index with delete, then re-creating the index with create. Indexes with a single field cannot be created.
  */
-export class DatabaseIndex extends pulumi.CustomResource {
+export class Index extends pulumi.CustomResource {
```

Comment:
It looks like this file was renamed to `indext_.ts`. Is that right?

Reply:
> It looks like this file was renamed to `indext_.ts`. Is that right?

This is expected due to https://github.com/pulumi/pulumi/blob/master/pkg/codegen/nodejs/gen.go#L200.